pkgs/applications/editors/kakoune/plugins/update.py

```diff
@@ -21,30 +21,40 @@
 )
 import pluginupdate
 
-GET_PLUGINS = f"""(with import <localpkgs> {{}};
+GET_PLUGINS = f"""with import <localpkgs> {{ }};
```

Comment:
This is the broken updater?

Reply:
Yes, if you run

```bash
cd pkgs/applications/editors/kakoune/plugins
nix-shell update-shell.nix
python update.py
```

It outputs (on master and here is the same error)
```nix
error:
       … <borked>

         at «none»:0: (source not available)

       … from call site

         at «string»:1:6:

            1| with import <localpkgs> { };
             |      ^
            2| let

       error: function 'anonymous lambda' called without required argument 'kakouneUtils'

       at /home/perchun/dev/nixpkgs/pkgs/applications/editors/kakoune/plugins/default.nix:1:1:

            1| { callPackage, config, kakouneUtils, lib }:
             | ^
            2|
```

Reply:
Thanks! It's on the list to go fix. I'm camping this weekend but will attend to it when back.